bindings/pyroot/src/Pythonize.cxx

```diff
@@ -2273,11 +2273,14 @@ namespace {
       Py_XDECREF(cppname);
 
       std::string pprint;
-      std::stringstream calcPrintValue;
-      calcPrintValue << "*((std::string*)" << &pprint << ") = cling::printValue((" << className << "*)"
-                     << self->GetObject() << ");";
-      gInterpreter->Calc(calcPrintValue.str().c_str());
-      return PyROOT_PyUnicode_FromString(pprint.c_str());
+      void *myObj = self->GetObject();
+      std::stringstream ss;
+      ss << myObj;
+      std::string code =  "*((" + className + "*)" + ss.str() + ")";
+      pprint = gInterpreter->Evaluate(code.c_str());
```

Comment:
You should capture a possible error from `Evaluate` and then handle it, somehow.